superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -373,6 +424,29 @@ const SliceHeaderControls = (props: SliceHeaderControlsPropsWithRouter) => {
     ? t('Exit fullscreen')
     : t('Enter fullscreen');
 
+  const [threadListId, setthreadListId] = useState<string | null>(null);
+  const [threadIsOpen, setThreadIsOpen] = useState<boolean>(false);
+  const [selectedThreadID, setSelectedThreadID] = useState<string | null>(null);
```

Comment:
@kgabryje I had originally intended for this UUID to be for a thread, not a threadList, so it may need to be renamed?